requirements/pytorch/base.txt

```diff
@@ -3,6 +3,7 @@
 
 numpy>=1.17.2, <1.23.1
 torch>=1.9.*, <1.13.0
+lightning-lite
```

Comment:
It should probably not be included here yet?

Reply:
It is for testing as you would need to include it in the first PR, which moves some functionality anyway... :)

Reply:
without it, the installation fails, because it's not on pypi. See your previous commit:

https://github.com/Lightning-AI/lightning/runs/8170542798?check_suite_focus=true

So what should we do?